urwid/widget/__init__.py

```diff
@@ -0,0 +1,170 @@
+from __future__ import annotations
+
+from .attr_map import AttrMap, AttrMapError
+from .attr_wrap import AttrWrap
+from .bar_graph import BarGraph, BarGraphError, BarGraphMeta, GraphVScale, scale_bar_values
+from .big_text import BigText
+from .box_adapter import BoxAdapter, BoxAdapterError
+from .columns import Columns, ColumnsError
+from .constants import (
+    RELATIVE_100,
+    Align,
+    Sizing,
+    VAlign,
+    WHSettings,
+    WrapMode,
+    normalize_align,
+    normalize_height,
+    normalize_valign,
+    normalize_width,
+    simplify_align,
+    simplify_height,
+    simplify_valign,
+    simplify_width,
+)
+from .container import WidgetContainerMixin
+from .divider import Divider
+from .edit import Edit, EditError, IntEdit
+from .filler import Filler, FillerError, calculate_top_bottom_filler
+from .frame import Frame, FrameError
+from .grid_flow import GridFlow, GridFlowError
+from .line_box import LineBox
+from .overlay import Overlay, OverlayError
+from .padding import Padding, PaddingError, calculate_left_right_padding
+from .pile import Pile, PileError
+from .popup import PopUpLauncher, PopUpTarget
+from .progress_bar import ProgressBar
+from .solid_fill import SolidFill
+from .text import Text, TextError
+from .widget import (
+    BoxWidget,
+    FixedWidget,
+    FlowWidget,
+    Widget,
+    WidgetError,
+    WidgetMeta,
+    WidgetWrap,
+    WidgetWrapError,
+    delegate_to_widget_mixin,
+    fixed_size,
+    nocache_widget_render,
+    nocache_widget_render_instance,
+)
+from .widget_decoration import WidgetDecoration, WidgetDisable, WidgetPlaceholder
+from .wimp import Button, CheckBox, CheckBoxError, RadioButton, SelectableIcon
+
+__all__ = (
```

Comment:
`__all__` needed due to backwards-compatibility constants definition (as it's not used locally, it's defined as "public" explicit)